src/platforms/php/guides/symfony/index.mdx

```diff
@@ -61,12 +61,24 @@ return [
 
 ### Monolog Integration
 
-If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the normal error listener approach, you also need these additional configuration and services to log the errors correctly:
+If you are using [Monolog](https://github.com/Seldaek/monolog) to report events instead of the normal error listener approach, you also need this additional configuration to log the errors correctly:
 
 ```yaml {filename:config/packages/sentry.yaml}
 sentry:
     register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
 
+monolog:
+    handlers:
+        sentry:
+            type: sentry
+            level: !php/const Monolog\Logger::ERROR
+            hub_id: Sentry\State\HubInterface
+```
+
+If you are using a version of [MonologBundle](https://github.com/symfony/monolog-bundle) below `3.7` you need to
```

Comment:
"Below" sounded a bit awkward to me, but do what makes sense here.

```suggestion
If you are using a version of [MonologBundle](https://github.com/symfony/monolog-bundle) before `3.7`, you need to
```

Reply:
@imatwawana "before" sounds better, the reason I used "below" was cause that word is used on line 22.